src/utilities/commentsPopUpUtilities.js

```diff
@@ -0,0 +1,65 @@
+import commentsCount from '../counterFunctions/commentsCounter.js';
+import { getCommentsData } from './fetchData.js';
+
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/kUJtIKt0WlDGnehZIL7s/comments';
+
+export const renderComments = (id, comments) => {
+  const commentTitle = document.querySelector('.comments-title');
+
+  getCommentsData(id, url)
+    .then((response) => response.json())
+    .then((data) => {
+      commentTitle.innerHTML = `Comments (${commentsCount(data) || 0})`;
+      comments.innerHTML = '';
+      data.forEach((comment) => {
+        const newComment = document.createElement('li');
+        newComment.innerHTML = `<span class="commment-date">(${comment.creation_date}) </span><span class="user-name">${comment.username}:</span> <span
+            class="comment-text">${comment.comment}</span>`;
+        comments.appendChild(newComment);
+      });
+    })
+    .catch((error) => error);
+};
+
+export const renderPopup = (obj) => {
+  const container = document.createElement('div');
+  container.classList.add('popup-container');
+
+  const popupCommentWindow = document.createElement('div');
+  popupCommentWindow.classList.add('popup');
+  popupCommentWindow.innerHTML = `
+    <button class="close-btn"><i class="fa-solid fa-xmark"></i></button>
+    <section class="show-summary">
+      <div class="shows-poster">
+        <img src="${obj.image.medium}" alt="show poster">
+      </div>
+      <div class="show-info">
+        <h2  class="show-title">${obj.name}</h2>
+        <ul class="genres">${obj.genres
+    .map((genre) => `<li class='genre-item'>${genre}</li>`)
+    .join('')}</ul>
+        <div class="rating">Rating: <span class="rating">${obj.rating.average}</span></div>
+        ${obj.summary}
+      </div>
+    </section>
+    <section class="add-comment-section">
+      <h2>Add comment: </h2>
+      <form class="popup-form" action="#">
+        <input type="text" class="name-input">
+        <textarea name="" id="" class="comment-input" rows="3"></textarea>
+        <button type="submit">Add comment</button>
+      </form>
+    </section>
+    <section class="comments-section">
+      <h2 class="comments-title">Comments: </h2>
+      <ul class="comments-list"></ul>
```

Comment:
- [ ] [OPTIONAL] your website looks awesome and I'm really impressed  👏  I would love to suggest a small CSS modification to your comments-list & reservation-list to improve the UI/UX , I would suggest adding a fixed height to your list and add the attribute `overflow-y: scroll;`
 example : 
```CSS
.comments-list {
    display: flex;
    flex-direction: column;
    gap: 10px;
    /* add fixed height or max-height*/
    height: 205px;
    /*add scroll bar to scroll through the list only*/
    overflow-y: scroll;
}

```
**_Result :_** 
![OVERFIX](https://user-images.githubusercontent.com/89282221/197286362-db9ffd55-66f8-4bc9-bf4b-7cbc7cf2bb01.gif)